src/app.js

```diff
@@ -0,0 +1,65 @@
+import NewsApiService from "./services/news.service";
+
+const API_KEY = "cd77286a82d24223b1320716435aacb6";
+
+export default class App {
+  constructor() {
+    this.service = new NewsApiService(API_KEY);
+  }
+
+  async init() {
+    this.sourceId = this.getQueryStringValue("sourceId");
+
+    let sources = await this.service.getSources();    
```

Comment:
should be `const` as there is no other assignment

Reply:
fixed